package.json

```diff
@@ -43,6 +43,13 @@
     "typescript": "4.1.3"
   },
   "peerDependencies": {
-    "eslint": "^7.5.0"
+    "eslint": "^7.5.0",
+    "eslint-config-prettier": "^8.1.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jest": "^24.3.4",
+    "eslint-plugin-jsdoc": "^32.3.0",
+    "@typescript-eslint/eslint-plugin": "^4.21.0",
+    "@typescript-eslint/parser": "^4.21.0",
```

Comment:
```json
"eslint-config-prettier": "8.3.0",
 "eslint-plugin-import": "2.24.2",
 "eslint-plugin-jest": "24.5.2",
 "eslint-plugin-jsdoc": "36.1.0",
"@typescript-eslint/eslint-plugin": "4.33.0",
 "@typescript-eslint/parser": "4.33.0",
```

Those versions are used in our project. We still have an issue with `eslint-plugin-jsdoc`, it's major version is different.

Just a thought.. should we include all of those dependencies as dependencies rather than peerDependencies? That way we wouldn't need to handle them on each project, just this one. What do you think?

Reply:
Changed peerDependency lock to anything lagger as in ngx-eds. 

I've keept the eslint as before (locked to ^7.5.0)

Latest 👇 
```
    "eslint": "^7.5.0",
    "eslint-config-prettier": ">=8.1.0",
    "eslint-plugin-import": ">=2.22.1",
    "eslint-plugin-jest": ">=24.3.4",
    "eslint-plugin-jsdoc": ">=32.3.0",
    "@typescript-eslint/eslint-plugin": ">=4.21.0",
    "@typescript-eslint/parser": ">=4.21.0",
    "@angular-eslint/template-parser": ">=2.0.2 <13.0.0"
  }
  ```